api/models.py

```diff
@@ -158,6 +155,31 @@ class Revision(BaseModel):
     )
 
 
+class DefaultTimeout:
+    """Helper to create default values for timeout fields"""
+
+    def __init__(self, hours=0, minutes=0, delta=None):
```

Comment:
```suggestion
    def __init__(self, hours=6, minutes=0, delta=None):
```

Reply:
The class could just have `delta`, the `hours` and `minutes` are to make it easier to use.  I guess I could just remove `delta` if it makes things harder to understand and we'll only have `hours` and `minutes` granularity.

But yes if nothing is provided then of course the timeout will be now.  However, the whole point of this class is to provide a default value.

Reply:
Yes, I think it is better to drop `delta` from this.

Reply:
Done